lua/nvim-treesitter-playground/internal.lua

```diff
@@ -264,6 +269,23 @@ function M.highlight_nodes(bufnr, nodes)
   end
 end
 
+function M.goto_node(bufnr)
+  local bufwin = vim.fn.win_findbuf(bufnr)[1]
+  if bufwin then
+    api.nvim_set_current_win(bufwin)
+  else
+    local node = M.get_current_node(bufnr)
+
+    local win = api.nvim_get_current_win()
+    vim.cmd "vsplit"
+    api.nvim_win_set_buf(win, bufnr)
+    api.nvim_set_current_win(win)
```

Comment:
This is written is a bit convoluted way to get the split in the other direction as the playground would open.